### PR TITLE
web/storagenode: logic removed from template

### PR DESCRIPTION
**Pull request by VitaliiShpital**
What: logic removed from templates, replaced by new computed properties

Why: due to vue style guide

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: web/storagenode/src/app/components/BarInfoContainer.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <p class="remaining-space-container__amount"><b>{{remaining}}GB</b></p>
         <div class="remaining-space-container__bar">
             <InfoComponent :text="infoMessage">
-                <Bar :current="currentBarAmount" :max="maxBarAmount" color="#224CA5"/>
+                <Bar :current="currentBarAmount" :max="maxBarAmount" color="#224CA5" />
```
Comment (stefanbenten) [inline web/storagenode/src/app/components/BarInfoContainer.vue]: unnecessary change?

File: web/storagenode/src/app/components/Bar.vue [hunk 2]
```diff
@@ -8,36 +8,36 @@
...
-     */
-    class BarFillStyle {
-        'background-color': string;
-        width: string;
-
-        public constructor(backgroundColor: string, width: string) {
-            this['background-color'] = backgroundColor;
-            this.width = width;
-        }
+import { Component, Prop, Vue } from 'vue-property-decorator';
```
Comment (stefanbenten) [inline web/storagenode/src/app/components/Bar.vue]: Why did this indentation got removed?

File: web/storagenode/src/app/components/PayoutContainer.vue [hunk 3]
```diff
@@ -18,15 +18,15 @@
 </template>
 
 <script lang="ts">
-    import { Component, Prop, Vue } from 'vue-property-decorator';
+import { Component, Prop, Vue } from 'vue-property-decorator';
```
Comment (stefanbenten) [inline web/storagenode/src/app/components/PayoutContainer.vue]: Same indentation problem here

File: web/storagenode/src/app/components/SNOFooter.vue [hunk 4]
```diff
@@ -14,10 +14,10 @@
 </template>
 
 <script lang="ts">
-    import { Component, Vue } from 'vue-property-decorator';
```
Comment (stefanbenten) [inline web/storagenode/src/app/components/SNOFooter.vue]: Indent Problem

File: web/storagenode/src/app/components/SatelliteSelectionContainer.vue [hunk 5]
```diff
@@ -7,7 +7,7 @@
         <svg width="8" height="4" viewBox="0 0 8 4" fill="none" xmlns="http://www.w3.org/2000/svg" alt="arrow image">
             <path d="M3.33657 3.73107C3.70296 4.09114 4.29941 4.08814 4.66237 3.73107L7.79796 0.650836C8.16435 0.291517 8.01864 0 7.47247 0L0.526407 0C-0.0197628 0 -0.16292 0.294525 0.200917 0.650836L3.33657 3.73107Z" fill="#535F77"/>
         </svg>
-        <SatelliteSelectionDropdown v-if="isPopupShown"/>
+        <SatelliteSelectionDropdown v-if="isPopupShown" />
```
Comment (stefanbenten) [inline web/storagenode/src/app/components/SatelliteSelectionContainer.vue]: unnecessary change

File: web/storagenode/src/app/components/SNOContentFilling.vue [hunk 6]
```diff
@@ -36,8 +36,8 @@
         <div v-if="selectedSatellite.id">
             <p class="info-area__title">Uptime & Audit Checks by Satellite</p>
             <div class="info-area__checks-area">
-                <ChecksAreaContainer label="Uptime Checks" :amount="checks.uptime" info-text="Uptime checks occur to make sure  your node is still online. This is the percentage of uptime checks you’ve passed."/>
-                <ChecksAreaContainer label="Audit Checks" :amount="checks.audit" info-text="Percentage of successful pings/communication between the node & satellite."/>
+                <ChecksAreaContainer label="Uptime Checks" :amount="checks.uptime" info-text="Uptime checks occur to make sure  your node is still online. This is the percentage of uptime checks you’ve passed." />
```
Comment (stefanbenten) [inline web/storagenode/src/app/components/SNOContentFilling.vue]: Unnecessary Changes

File: web/storagenode/src/app/components/Bar.vue (hunk 2 above)
Comment (VitaliiShpital) [reply web/storagenode/src/app/components/Bar.vue]: https://vuejs.org/v2/style-guide/?#Non-flux-state-management-use-with-caution

due to vue style guide indents inside script tag are not used. According to that new linter rule was added for such purposes earlier. 

Actually indents are not used inside style tag too, but fixing that would make PR too big. Should we decide to follow this or leave it be?

Comment (stefanbenten) [reply web/storagenode/src/app/components/Bar.vue]: I think making a different PR to fix the styling issues makes sense 👍
